index.js

```diff
@@ -69,7 +69,7 @@ async function boot() {
             redisInstance: redis,
             buildId: job.args[0].buildId,
             status: 'FAILURE',
-            statusMessage: 'Build failed to start due to infrastructure error'
+            statusMessage: `${failure}`
```

Comment:
When a pod is deleted some problems, the error `Build failed to start due to infrastructure error` will never shown by this change. `Circuitbreaker timeout` will shown alternatively.

Reply:
I've fixed this.